cmd/cloud/utils.go

```diff
@@ -51,6 +51,14 @@ func registerPagingFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("include-deleted", false, "Whether to include deleted objects.")
 }
 
+func getPagingModel(pf pagingFlags) model.Paging {
```

Comment:
nit: `getPaging`, mostly because we're not using `model` suffix anywhere else, though it's not very confusing. 0/5

Reply:
Right. I will rename it to `getPagingOption`. What do you think?

Reply:
0/5. Whatever you think its best. 👍

Reply:
I also like `getPaging`, but only 1/5 on it.